neurodsp/sim/cycles.py

```diff
@@ -204,6 +204,109 @@ def sim_gaussian_cycle(n_seconds, fs, std):
     return cycle
 
 
+def sim_gaussian_asym_cycle(n_seconds, fs, beta, scale=2, shift=1):
```

Comment:
we need to change the name on this, I think. It ends 'gaussian-like', but that's not how it's generated. I'm not sure what the best name is though...

Reply:
Maybe ``sim_exp_asym_cycle`` or `sim_beta_asym_cycle`` since it's being raised to beta?

Reply:
Updated to `sim_asym_beta_cycle` and `sim_asym_harmonic_cycle`. Feel free to change.